eth2util/network.go

```diff
@@ -50,10 +50,16 @@ var (
 		GenesisForkVersionHex: "0x90000069",
 		GenesisTimestamp:      1655733600,
 	}
+	Holesky = Network{
+		ChainID:               17000,
+		Name:                  "holesky",
+		GenesisForkVersionHex: "0x00017000",
+		GenesisTimestamp:      1694786400,
```

Comment:
Is this time changing since launch was delayed?

Reply:
I took these data from the ticket itself. 

Didn't verify it myself

Reply:
And yeah, i think the timestamp would change to reflect next week's

Reply:
They haven't updated the README yet - https://github.com/eth-clients/holesky#metadata

Reply:
According to [this pr](https://github.com/eth-clients/holesky/pull/73/files) it is `1696000704`

Reply:
yeah cool, let's merge this PR only when the [linked PR](https://github.com/eth-clients/holesky/pull/73/files) is merged